Makefile

```diff
@@ -119,6 +119,10 @@ gogci:
 gotidy:
 	$(MAKE) $(FOR_GROUP_TARGET) TARGET="tidy"
 
+.PHONY: topotidy
+topotidy:
+	@python ./internal/buildscripts/topotidy.py
```

Comment:
I don't think we can expect devs will have python installed

Reply:
Yes, Pablo also told me Collector devs might not know Python, which would make maintenance problematic, so I will probably rewrite this script in Go. I opened this PR to get the code out there, but I'll probably close it and open a new one when things are more fleshed out.

Reply:
I rewrote the script in Go and opened a PR on `opentelemetry-go-build-tools` to add it as a `crosslink` subcommand if you want to take a look: https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/642